### ISSUE TITLE
Save as feature

### ISSUE THREAD

**Issue by Stephen-Gates** [ODIQueensland/data-curator#625]
### Desired Behaviour 

Similar to the open file #596, simplify the [save as](https://app.cucumber.pro/projects/data-curator/documents/branch/develop/test/features/file/save-as.feature) menu to remove the need to specify the type of file you want to save

**Comment by pripley123** [comment]
I would be great if I could save a data package that still had errors i.e., a data package work in progress.  It's frustrating when you add metadata and then get an error that's something needs fixing, but then can't save your work... especially when it's not clear where the error is.... perhaps this would be less of an issue if I could more easily find the source of errors

**Comment by Stephen-Gates** [comment]
Hi @pripley123 does https://github.com/ODIQueensland/data-curator/issues/265 describe what you're looking for? I'd be interested to know in what situations it's not clear where the error is - perhaps we can improve that also.

**Comment by pripley123** [comment]
Yes, looks like 265 would address the data package saving issue.


As for as error messages go... I seem to get "There is a required table or column property that is missing. Please check that all required properties are entered" fairly often... this is frustrating since there's no indication of what field / table is causing the issue.   Then I have to inspect all the fields / tables to try to find out what is causing it.  It would be great if it said, there's an issue with field X of table Y e.g., the name field of table xyz is missing.  It would be even better if there was a link so I could jump directly to the field that was causing the issue.


Hope that helps,
Paul

________________________________

**Comment by Stephen-Gates** [comment]
Thanks Paul, 

To export a data package, the following is required for each table:

- header row (first row) and second row of data
- use the Check Header Row menu item to set all Column names 
- set a table name
- data saved

Then for the data package:
- enter some provenance information
- set a package name

The error you're seeing is a catch all, so I'm not sure what you're missing. If you work it out, let me know and I'll improve the error message

#265 has been moved to v0.17.0 for the Sponsor's consideration so it may be fixed earlier depending on priorities and budget
